ui/tests/utils/index.js

```diff
@@ -0,0 +1,11 @@
+import { mount, shallowMount } from '@vue/test-utils';
+import merge from 'lodash.merge';
+
+export const createFactory =
```

Comment:
what would be the goal of this factory?, it is mounting and setting options

Reply:
It's a way to simplify component setup. When you need to set up lots of properties for a component, like in the [FormDialog test](https://github.com/cloudblue/connect-bi-reporter/pull/31/files/784e77f0e4aab86832933d5f095b1000d57b197e#diff-31180ca045168e5af3e2740c661222742c33c6ab0eb90279e008b82340575d9bR6), you need to repeat that in each point where you mount the component, which is very painful.

This is a pattern that I've used in the past with Vue Test Utils, it works quite well for medium-to-large components, or just small components that need lots of stubs.

I am not sure if this is the best option to use as a standard for our extension tests, but I'd keep it for now since it solves an issue, and we can figure out if we like it or not during development. Wdyt?